### DISCUSSION TITLE
php artisan horizon:forget

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#8320] [General]
Hi

When using `horizon:forget` command like this
```
php artisan horizon:forget 25d8d9d1-56f4-4d75-a93e-dydgac0b45ae
```
I see the job is removed from Horizon Failed Jobs list

But, I get this error in Terminal

```
SQLSTATE[HY000] [2002] Connection refused (Connection: mysql, SQL: delete from `failed_jobs` where `uuid` = 25d8d9d1-56f4-4d75-a93e-dydgac0b45ae)

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:795
    791▕         // If an exception occurs when attempting to run a query, we'll format the error
    792▕         // message to include the bindings with SQL, which will make this exception a
    793▕         // lot more helpful to the developer instead of just the database's errors.
    794▕         catch (Exception $e) {
  ➜ 795▕             throw new QueryException(
    796▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    797▕             );
    798▕         }
    799▕     }

      +30 vendor frames 

  31  artisan:35
      Illuminate\Foundation\Console\Kernel::handle()
```

I got a similar message in my laravel.log when uploading images and using glide presets. I posted about that problem last week.
https://github.com/statamic/cms/discussions/8287

**Comment by duncanmcclean** [comment]
It's trying to connect to a database (because that's where failed jobs usually live) but as you probably don't have a database, it errors out.

[This addon](https://statamic.com/addons/jonassiewertsen/jobs) might be helpful for storing failed jobs if you're not using a database - I've never used it myself.

**Reply by stephenmeehanuk** [reply]
Ah. I see. Thanks. I don't normally worry about removing failed jobs, as the list clears itself.

I'll give that addon a go, once I've managed to figure out what's triggering the other problem. https://github.com/statamic/cms/discussions/8287
